package.json

```diff
@@ -37,8 +37,8 @@
     "html-webpack-plugin": "^4.0.0-beta.4",
     "jasmine-ajax": "^3.4.0",
     "jasmine-core": "^2.8.0",
-    "jquery": "~3.3.1",
-    "karma": "^3.0.0",
+    "jquery": "^3.4.1",
```

Comment:
This is only used for tests, because our jQuery dependency is actually provided by REDCap at runtime. Better safe than sorry, though.